ui/app/AppLayouts/Chat/panels/communities/CommunityMembersTabPanel.qml

```diff
@@ -0,0 +1,115 @@
+import QtQuick 2.14
+import QtQuick.Layouts 1.14
+import QtQuick.Controls 2.14
+
+import StatusQ.Core 0.1
+import StatusQ.Core.Theme 0.1
+import StatusQ.Controls 0.1
+import StatusQ.Components 0.1
+import StatusQ.Popups 0.1
+
+import utils 1.0
+import shared.controls.chat 1.0
+
+import "../../layouts"
+
+Item {
+    id: root
+
+    property string placeholderText
+    property var model
+
+    signal userProfileClicked(string id)
+    signal kickUserClicked(string id, string name)
+    signal banUserClicked(string id, string name)
+    signal unbanUserClicked(string id)
+
+    enum TabType {
+        AllMembers,
+        BannedMembers
+    }
+
+    property int panelType: CommunityMembersTabPanel.TabType.AllMembers
```

Comment:
Just a nitpick: I think `panelType` is not a good name here because the panel in itself behaves the same (apart from rendering different buttons). I'd suggest `memberType` instead. 

Again just a suggestion, no biggie if you disagree.

Reply:
I would rather name it `activeTab`

Reply:
I will leave it as it is for now. In next ticket I am going to add new tabs: Pending Requests and Rejected Requests. So , memberType will not be a suitable name. Re activeTab, this property indicates the type of tab, not activity state. Maybe I will come up with some better&suitable name within a next ticket.